app/src/features/authentication/signup/SignupForm.tsx

```diff
@@ -0,0 +1,166 @@
+import { useForm, SubmitHandler } from "react-hook-form";
```

Comment:
it's a personal preference but i think it makes the imports more organized to keep all the external ones (like the ones from react, 3rd party libraries) at the top then custom hooks, then components